pkg/registry/core/pod/rest/log.go

```diff
@@ -86,7 +88,10 @@ func (r *LogREST) Get(ctx context.Context, name string, opts runtime.Object) (ru
 
 	countSkipTLSMetric(logOpts.InsecureSkipTLSVerifyBackend)
 
-	if errs := validation.ValidatePodLogOptions(logOpts); len(errs) > 0 {
+	if !utilfeature.DefaultFeatureGate.Enabled(features.PodLogsQuerySplitStreams) {
+		logOpts.Stream = nil
+	}
+	if errs := validation.ValidatePodLogOptions(logOpts, utilfeature.DefaultFeatureGate.Enabled(features.PodLogsQuerySplitStreams)); len(errs) > 0 {
```

Comment:
This seems to break the tests.

Reply:
not just the tests, I think it breaks existing pod log API clients when the feature is enabled